### ISSUE TITLE
Characterise Hazard missing in Study 1

### ISSUE THREAD

**Issue by p-a-s-c-a-l** [clarity-h2020/docker-drupal#50]
Somehow step 1 'Characterise Hazard' disappeared from study 1:

![grafik](https://user-images.githubusercontent.com/1788226/53511891-b74eba80-3ac1-11e9-97a0-a3884a64c160.png)

Under [Related entities for Adaptation Scenarios for Metropolitan Resilience Planning](https://csis.myclimateservice.eu/group/1/content) it is still shown but [Study Group main navigation v2 (Taxonomy term)](https://csis.myclimateservice.eu/admin/structure/views/view/study_group_main_navigation_v2/edit/block_1) shows the following error:

``
SQLSTATE[22P02]: Invalid text representation: 7 ERROR: invalid input syntax for integer: "structure" LINE 9: ... ((group_content_field_data_node_field_data.gid = 'structure... ^: SELECT taxonomy_term__field_study_step.field_study_step_value AS taxonomy_term__field_study_step_field_study_step_value, taxonomy_term_field_data.tid AS tid, field_field_eu_gl_methodology_taxonomy_term_field_data.nid AS field_field_eu_gl_methodology_taxonomy_term_field_data_nid, group_content_field_data_node_field_data.id AS group_content_field_data_node_field_data_id, groups_field_data_group_content_field_data.id AS groups_field_data_group_content_field_data_id FROM {taxonomy_term_field_data} taxonomy_term_field_data LEFT JOIN {node__field_field_eu_gl_methodology} node__field_field_eu_gl_methodology ON taxonomy_term_field_data.tid = node__field_field_eu_gl_methodology.field_field_eu_gl_methodology_target_id AND node__field_field_eu_gl_methodology.deleted = :views_join_condition_0 LEFT JOIN {node_field_data} field_field_eu_gl_methodology_taxonomy_term_field_data ON node__field_field_eu_gl_methodology.entity_id = field_field_eu_gl_methodology_taxonomy_term_field_data.nid LEFT JOIN {group_content_field_data} group_content_field_data_node_field_data ON field_field_eu_gl_methodology_taxonomy_term_field_data.nid = group_content_field_data_node_field_data.entity_id AND group_content_field_data_node_field_data.type = :views_join_condition_1 LEFT JOIN {groups_field_data} groups_field_data_group_content_field_data ON group_content_field_data_node_field_data.gid = groups_field_data_group_content_field_data.id LEFT JOIN {taxonomy_term__field_study_step} taxonomy_term__field_study_step ON taxonomy_term_field_data.tid = taxonomy_term__field_study_step.entity_id AND taxonomy_term__field_study_step.deleted = :views_join_condition_2 WHERE ((group_content_field_data_node_field_data.gid = :group_content_field_data_gid)) AND ((taxonomy_term_field_data.status = :db_condition_placeholder_3) AND (taxonomy_term_field_data.vid IN (:db_condition_placeholder_4))) ORDER BY taxonomy_term__field_study_step_field_study_step_value ASC NULLS FIRST LIMIT 20 OFFSET 0; Array ( [:group_content_field_data_gid] => structure [:db_condition_placeholder_3] => 1 [:db_condition_placeholder_4] => eu_gl [:views_join_condition_0] => 0 [:views_join_condition_1] => study-group_node-gl_step [:views_join_condition_2] => 0 ) 
``

**Comment by DanielRodera** [comment]
I was taking a look to this issue and I realized the "Hazard Characterization" taxonomy also had disappeared... 
Maybe this is causing the problem, should I create the taxonomy again?

**Comment by p-a-s-c-a-l** [comment]
> Maybe this is causing the problem, should I create the taxonomy again?

yes, please recreate it. thanks!

**Comment by DanielRodera** [comment]
> > Maybe this is causing the problem, should I create the taxonomy again?
> 
> yes, please recreate it. thanks!

Taxonomy cretaed, seems the problem is still not solved.

**Comment by patrickkaleta** [comment]
> > > Maybe this is causing the problem, should I create the taxonomy again?
> > 
> > 
> > yes, please recreate it. thanks!
> 
> Taxonomy cretaed, seems the problem is still not solved.

I just checked this and the problem is indeed that the taxonomy term has been removed. This caused that all GL-Step entities which referenced this term, lost this reference information and on this referenec information the tab navigation is build upon.

Re-creating this lost term therefore doesn't solve this problem alone, since now we have to re-create the lost references between this term and the GL-step entitities. I've already done this for the first study and the tab is now shown again as before.

We have to be careful with deleting taxonomy terms (especially in the EU-GL taxonomy)
